packages/yew-macro/tests/hook_macro/use_prepared_state-fail.stderr

```diff
@@ -1,14 +1,16 @@
-error: expected closure
- --> tests/hook_macro/use_prepared_state-fail.rs:6:38
+error: this hook takes 2 arguments but 1 argument was supplied
+ --> tests/hook_macro/use_prepared_state-fail.rs:6:5
   |
 6 |     use_prepared_state_with_closure!(123)?;
-  |                                      ^^^
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This test is trying test that the second argument is expected to be a closure. It should have a second argument that is not closure.

Reply:
There's another test for that